metadata/org.postgresql/postgresql/index.json

```diff
@@ -1,10 +1,10 @@
 [
   {
     "latest": true,
```

Comment:
This needs to be changed. We do not want to replace support for `42.3.4` with `42.7.3` but instead add support for `42.7.3` in addition to `42.3.4`. See https://github.com/oracle/graalvm-reachability-metadata/blob/master/metadata/com.github.ben-manes.caffeine/caffeine/index.json for inspiration how to have more than one major version of a library supported.